src/data/sym/basic.lean

```diff
@@ -244,10 +244,11 @@ begin
 end
 
 /-- If α ≃ β then sym α n ≃ sym β n. -/
```

Comment:
```suggestion
/-- If `α ≃ β` then `sym α n ≃ sym β n`. -/
```
To be even more precise, you could instead say
```suggestion
/-- Constructs an equivalence `sym α n ≃ sym β n` from an equivalence `α ≃ β`. -/
```
as `α ≃ β` is not a Prop, but data

Reply:
Neither of these docstrings says anything that's not already in the type! It should specify that it acts elementwise like `sym.map`.

Reply:
I copied [a similar docstring](https://leanprover-community.github.io/mathlib_docs/data/equiv/basic.html#equiv.equiv_congr) in mathlib and adjusted it. If you are not happy with that, please let me know :joy: That docstring also doesn't say anything that's not in the type, so I guess if my docstring is unacceptable then the docstring I copied from must be fixed too :stuck_out_tongue: